src/test/java/org/broadinstitute/hellbender/utils/test/HelpSmokeTest.java

```diff
@@ -2,6 +2,7 @@
 
 import org.broadinstitute.hellbender.CommandLineProgramTest;
 import org.broadinstitute.hellbender.utils.help.GATKHelpDoclet;
+import org.broadinstitute.hellbender.utils.runtime.ProcessController;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
```

Comment:
Maybe this class should be called something like `DocGenIntegrationTest` instead of `HelpSmokeTest`, which is a little puzzling/inconsistent with our test class naming conventions.